jib-cli/src/main/java/com/google/cloud/tools/jib/cli/CommonCliOptions.java

```diff
@@ -244,6 +244,15 @@
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private boolean serialize;
 
+  @CommandLine.Option(
+      names = "--image-json",
+      paramLabel = "<path-to-json>",
+      description =
+          "A path to the destination directory of jib-image.json. If provided, image details (for example, digest, id"
```

Comment:
Why not accepting the whole path but a directory? I think it can be restrictive to some users who want to have a different name.

And probably it should have `scope = CommandLine.ScopeType.INHERIT` like others?